pom.xml

```diff
@@ -43,14 +31,25 @@
                 <updatePolicy>always</updatePolicy>
             </snapshots>
         </repository>
+        <repository>
+            <id>jx-releases</id>
+            <url>https://nexus-jx.apps.serv.run/repository/maven-releases/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
     </repositories>
     <properties>
-        <maven.release.repository.id>sonatype-nexus-stage</maven.release.repository.id>
-        <maven.snapshot.repository.id>sonatype-nexus-snapshot</maven.snapshot.repository.id>
-        <maven.snapshot.repository.url>https://oss.sonatype.org/content/repositories/snapshots
-        </maven.snapshot.repository.url>
-        <maven.release.repository.url>https://oss.sonatype.org/service/local/staging/deploy/maven2/
-        </maven.release.repository.url>
+        <entando-engine.version>6.2.9</entando-engine.version>
+        <entando-admin-console.version>6.2.10</entando-admin-console.version>
+        <entando-portal-ui.version>6.2.4</entando-portal-ui.version>
+        <entando-plugin-jacms.version>6.2.8</entando-plugin-jacms.version>
+        <entando-plugin-jpversioning.version>6.2.3</entando-plugin-jpversioning.version>
+        <entando-plugin-jpseo.version>6.2.2</entando-plugin-jpseo.version>
+        <entando.keycloak.plugin.version>6.2.0-SNAPSHOT</entando.keycloak.plugin.version>
```

Comment:
Is this correct?

Reply:
Yes. Assuming your are referring to the keycloak SNAPSHOT  that pipeline still needs to be done